### PR TITLE
PatternFly milestone update

### PR DESCRIPTION
**Pull request by dlabrecq**
Update PatternFly and @redhat-cloud-services packages to the latest milestone.

https://issues.redhat.com/browse/COST-2503

### CODE REVIEW

**Review by dlabrecq — COMMENTED** [review]
Wait for official PatternFly release before merging

**Review by gitdallas — APPROVED** [review]
tested updates locally, didn't notice any issues

**Pull request merged**
